PauloAlves8039/angular-parking-frontend
# [FEATURE] Created alert service and refactored dropdown in component stay

### Context
This Pull Request includes the addition of customized alerts and refactoring of the customers and vehicles dropdwon.

### Changes 
  - **Notification Service:** Created `notification.service.ts`
  - **All Components:** Changed `StayTableComponent`, `AddressTableComponent`, `CustomerTableComponent`,  `CustomerVehicleTableComponent`,`VehicleTableComponent`.

### How to Test
  1. Navigate to the all components feature in the application.
  2. Verify that the all tables is correctly populated.
  3. Test the save, update, and delete operations via the modals.
  4. Ensure that all alerts work as expected.
  
### Notes
  - Ensure that all functionalities receive notifications.
  - Review the behavior of notifications.
  
### Related Issues
  - [Link to related issues or tasks, if applicable.]
